packages/front-end/pages/index.tsx

```diff
@@ -40,7 +40,10 @@ export default function Home(): React.ReactElement {
     } else if (hasExperiments) {
       router.replace("/experiments");
     } else {
-      router.replace("/getstarted");
+      // TODO: Feature flag here to split users between new setup flow and get started
+      // Gate to the org admin only
+      router.replace("/setup");
```

Comment:
Maybe we should also check if the setup has been completed to determine whether or not to take them here? They could have completed the setup but not have setup any experiments or feature flags so this would just take them to the setup complete page